media/js/src/assetDetail/AssetDetail.jsx

```diff
@@ -525,7 +566,7 @@ export default class AssetDetail extends React.Component {
                                     <div className="col-md-9">
                                         <div className="input-group">
 
-                                            {this.state.tab === 'createSelection' && (
+                                            {(this.state.tab === 'createSelection' || this.state.isEditing) && (
```

Comment:
The selection start & selection end buttons & timecode spinners show correctly in edit mode. But, the textual range display is also showing up. Looking at line 585, maybe a `!isEditing` needs to be added?